dts/bindings/sensor/nxp,tmpsns.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Polytech A/S
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP TMPSNS
+
+compatible: "nxp,tmpsns"
+
+include: sensor-device.yaml
+
+properties:
+  clock-divider:
+    type: int
+    default: 3
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules